src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/GetActionPlanTest.kt

```diff
@@ -122,6 +135,8 @@ class GetActionPlanTest : IntegrationTestBase() {
     val actual = response.responseBody.blockFirst()
     assertThat(actual)
       .isForPrisonNumber(prisonNumber)
+      .hasReviewDateCategory(ReviewDateCategory.SIX_MONTHS)
+      .hasNoReviewDate()
```

Comment:
The controversial bit! I'm not sure if I should automatically populate `reviewDate` if the `reviewDateCategory` is set to 3, 6 or 12 months 🤔 I've chosen not to as there are no requirements around this, but I'll try and find out more from the BAs.

Reply:
mmm...the more I think about it, the more I think I should set the `reviewDate`, as I can't see any downsides to this approach. We'll still know which option the user selected (i.e. to populate the UI, or for reporting purposes).

In contrast, I can foresee problems with not setting it! 😂

Reply:
Hmmm, I'm not so sure
My thinking is that you currently have it correct, and your test is perfect as its our safety net for and and when we might get asked to change the behaviour.

If the business tell us to calculate the `reviewDate` from the value of the enum, then thats fine. But I dont think we should 2nd guess that as a requirement. I think we should only do it if its documented / communicated as a business requirement and expected behaviour.

Reply:
Let me talk you through this alternative approach: https://github.com/ministryofjustice/hmpps-education-and-work-plan-api/pull/66